### ISSUE TITLE
Sharing 

### ISSUE THREAD

**Issue by RohollahHS** [exitudio/MMM#13]
Hi Ekkasit,
Can you please share the event files (TensorBoard logs) for training your text-to-motion transformer? I want to compare its accuracy and loss with those of my transformer to see how well my model is working. 
I know that you shared your logs for FID and Diversity metrics for your transformer here https://github.com/exitudio/MMM/issues/7#issuecomment-2066086402, but I need your transformer's accuracy and loss during training.
Thanks a lot.

**Comment by exitudio** [comment]
Hi,
You can find the training logs for the 1st, 2nd stages, and evaluation in the output folder if you have already downloaded them from [2.3. Pre-trained models]. Alternatively, you can download them manually from [this link](https://drive.google.com/file/d/1y7N6GgnwLe9HF02-FplXVA7TZ6zJZRUV/view).

P.S. I reran the models and achieved a better FID score than the one reported in the paper. The updated pre-trained model was uploaded on June 10, 2024. (Run the script in 2.3 to get the updated models).
